docs/api/status.md

```diff
@@ -88,7 +88,7 @@ It appears when:
 
 - There is out-of-memory in gRPC payload, or networks, or etc.
 - There are some server overload situations.
-- The gRPC message size is more extensive than the limit (default is 4MB).
+- The Sent or received message is larger than configured limit (default is 4MB).
```

Comment:
I think the wording `Sent` should be `sent`

Reply:
fixed